README.rst

```diff
@@ -73,6 +73,18 @@ If you wish to set a threshold for iviewer that is *lower* than for the server:
 NB: Z-projection is not supported for tiled images in OMERO
 (Images larger than 2048 * 2048 pixels per plane are tiled in iviewer).
 
+OMERO uses Spectrum Color Picker for selecting ROI colors. 
+Define rows with brackets, and use commas to seperate values. 
+You can only view the first item of each row without enabling show_palette_only.
+To define a color palette use::
+    
+    $ omero config set omero.web.iviewer.color_palette [rgb(0,0,0),blue],[#000000]
```

Comment:
The description here doesn't give me much info on what this setting does. After the first statement you need something like, "The `color_palette` setting allows you to specify a grid of colors for users to choose for ROIs".
Typo "separate". The config name doesn't include 'roi' anywhere. I wonder if a better name might be `roi_colors` or `roi_palette` or even `roi_color_palette`?